.gitignore

```diff
@@ -40,7 +40,6 @@ __pycache__
 emscripten_build/
 /docs/_build
 /docs/_static/robots.txt
-/deps
```

Comment:
Should we keep some patterns for evmone excluded here?

Reply:
I think its not really needed - it might be only there if someone is using windows and installed it via `scripts/install_evmone.ps1`..